forms-flow-documents/src/formsflow_documents/config.py

```diff
@@ -84,6 +84,9 @@ class _Config:  # pylint: disable=too-few-public-methods
         str(os.getenv("MULTI_TENANCY_ENABLED", default="false")).lower() == "true"
     )
 
+    # Configure LOG
+    CONFIGURE_LOGS = str(os.getenv("CONFIGURE_LOGS", default="true")).lower() == "true"
```

Comment:
Is it necessary to include the given environment variable in the sample.env file?

Reply:
Done